### ISSUE TITLE
Getting Started - no luck

### ISSUE THREAD

**Issue by brettswift** [haugene/docker-transmission-openvpn#275]
I'm on OSX.  Trying to get my IPVanish connected and download a torrent. 

run command:   I have a little bash script to help: 

```
#!/bin/bash
set -x

echo -n Password:
read -s password
echo

echo "Starting Docker container"
OPENVPN_PROVIDER=IPVANISH
OPENVPN_USERNAME=myemail@redacted.com
OPENVPN_PASSWORD=$password

LOCAL_STORAGE_PATH=/Users/bswift/torrents/downloads/


docker run --cap-add=NET_ADMIN --device=/dev/net/tun -d \
              -v ${LOCAL_STORAGE_PATH}:/data \
              -v /etc/localtime:/etc/localtime:ro \
              -e "OPENVPN_PROVIDER=${OPENVPN_PROVIDER}" \
              -e "OPENVPN_CONFIG=ipvanish-CA-Vancouver-yvr-c02" \
              -e "OPENVPN_USERNAME=${OPENVPN_USERNAME}" \
              -e "OPENVPN_PASSWORD=${OPENVPN_PASSWORD}" \
              -e "LOCAL_NETWORK=192.168.0.0/24" \
              -p 9091:9091 \
              haugene/transmission-openvpn

```

logs
```
 tail -f transmission.log
[2017-07-16 16:49:14.740] RPC Server Serving RPC and Web requests on port 127.0.0.1:9091/transmission/ (rpc-server.c:1110)
[2017-07-16 16:49:14.740] Port Forwarding Stopped (port-forwarding.c:180)
[2017-07-16 16:49:14.740] UDP Failed to set receive buffer: requested 4194304, got 425984 (tr-udp.c:84)
[2017-07-16 16:49:14.740] UDP Please add the line "net.core.rmem_max = 4194304" to /etc/sysctl.conf (tr-udp.c:89)
[2017-07-16 16:49:14.740] UDP Failed to set send buffer: requested 1048576, got 425984 (tr-udp.c:95)
[2017-07-16 16:49:14.740] UDP Please add the line "net.core.wmem_max = 1048576" to /etc/sysctl.conf (tr-udp.c:100)
[2017-07-16 16:49:14.740] DHT Generating new id (tr-dht.c:311)
[2017-07-16 16:49:14.740] Using settings from "/data/transmission-home" (daemon.c:528)
[2017-07-16 16:49:14.741] Saved "/data/transmission-home/settings.json" (variant.c:1266)
[2017-07-16 16:49:14.741] Watching "/data/watch" for new .torrent files (daemon.c:573)
[2017-07-16 16:49:48.738] DHT Attempting bootstrap from dht.transmissionbt.com (tr-dht.c:249)
[2017-07-16 16:50:08.738] DHT dht.transmissionbt.com:6881: Temporary failure in name resolution (tr-dht.c:128)
[2017-07-16 16:50:57.738] DHT dht.transmissionbt.com:6881: Temporary failure in name resolution (tr-dht.c:128)
[2017-07-16 16:51:57.738] DHT dht.transmissionbt.com:6881: Temporary failure in name resolution (tr-dht.c:128)
```

docker log:

```

Sun Jul 16 17:35:05 2017 Data Channel Encrypt: Cipher 'AES-256-CBC' initialized with 256 bit key
Sun Jul 16 17:35:05 2017 Data Channel Encrypt: Using 256 bit message hash 'SHA256' for HMAC authentication
Sun Jul 16 17:35:05 2017 Data Channel Decrypt: Cipher 'AES-256-CBC' initialized with 256 bit key
Sun Jul 16 17:35:05 2017 Data Channel Decrypt: Using 256 bit message hash 'SHA256' for HMAC authentication
Sun Jul 16 17:35:05 2017 Control Channel: TLSv1, cipher TLSv1/SSLv3 DHE-RSA-AES256-SHA, 2048 bit RSA
Sun Jul 16 17:35:05 2017 [ams-a01.ipvanish.com] Peer Connection Initiated with [AF_INET]81.171.98.2:443
Sun Jul 16 17:35:07 2017 SENT CONTROL [ams-a01.ipvanish.com]: 'PUSH_REQUEST' (status=1)
Sun Jul 16 17:35:08 2017 AUTH: Received control message: AUTH_FAILED
Sun Jul 16 17:35:08 2017 SIGTERM[soft,auth-failure] received, process exiting

```

Not sure if I'm doing something wrong here..   I can't seem to find any steps I'd require in the docs..  

If I drop a file into the downloads/watch directory (had to create the directory.. README implies the container creates it but it doesn't)..   nothing is picked up and I don't see any activity. 

What other logs can I share from the container?

**Comment by brettswift** [comment]
well I held out asking a question as I was getting stumped.. and then I fixed it.  :)

I'm not sure if the DNS resolution is required but I have that in my config in my trial attempts. 

I was sure it wasn't my password.. but yea.. the auth was failing. 

The one thing it doesn't do is create the watch directory.   I created the watch directory and dropped a torrent in there, when the container was failing.   So I killed the container and rebuilt it - but it didn't seem to pick up on the file..  

I nuked the whole transmission directory and contents of the watch dir,  then restarted the container and my torrent is on it's way. 

Awesome project.. thanks!
